packages/fxa-payments-server/src/routes/ProductV2/SubscriptionCreate/index.tsx

```diff
@@ -0,0 +1,472 @@
+              <h2>Set up your subscription</h2>
+            </Localized>
+            <Localized id="sub-guarantee">
+              <p className="subheading">30-day money-back guarantee</p>
+            </Localized>
+          </div>
+
+          <h3 className="billing-title">
+            <Localized id="sub-update-title">
+              <span className="title">Billing Information</span>
+            </Localized>
+          </h3>
+
+          <AcceptedCards />
+
+          <ErrorMessage isVisible={!!paymentError.error}>
+            {paymentError.error && (
+              <Localized id={getErrorMessage(paymentError.type)}>
+                <p data-testid="error-payment-submission">
+                  {getErrorMessage(paymentError.type)}
+                </p>
+              </Localized>
+            )}
+          </ErrorMessage>
+
+          <ErrorMessage isVisible={isCardError}>
+            <Localized id={getErrorMessage('card_error')}>
+              <p data-testid="error-card-rejected">
+                {getErrorMessage('card_error')}
+              </p>
+            </Localized>
+          </ErrorMessage>
+
+          <ErrorMessage isVisible={!!createCustomerState.error}>
+            {createCustomerState.error ? (
+              <p data-testid="error-customer-status">
+                {createCustomerState.error.message}
+              </p>
+            ) : null}
+          </ErrorMessage>
+
+          <ErrorMessage
+            isVisible={!!(createSubscriptionState.error && !isCardError)}
+          >
+            {createSubscriptionState.error ? (
+              <p data-testid="error-sub-status">
+                {createSubscriptionState.error.message}
+              </p>
+            ) : null}
+          </ErrorMessage>
+
+          <PaymentForm
+            {...{
+              submitNonce,
+              onSubmit,
+              onChange,
+              inProgress,
+              validatorInitialState,
+              confirm: true,
+              plan: selectedPlan,
+              onMounted: onFormMounted,
+              onEngaged: onFormEngaged,
+            }}
+          />
+        </div>
+        <PlanDetails
+          {...{
+            profile,
+            selectedPlan,
+            isMobile,
+            showExpandButton: isMobile,
+          }}
+        />
+        <MobileCreateHeading {...{ isMobile }} />
+      </div>
+    </>
+  );
+};
+
+const MobileCreateHeading = ({ isMobile }: { isMobile: boolean }) =>
+  isMobile ? (
+    <div className="mobile-subscription-create-heading">
+      <div className="subscription-create-heading">
+        <Localized id="product-plan-details-heading">
+          <h2>Set up your subscription</h2>
+        </Localized>
+        <Localized id="sub-guarantee">
+          <p className="subheading">30-day money-back guarantee</p>
+        </Localized>
+      </div>
+    </div>
+  ) : null;
+
+export const handleSubscriptionPayment = async ({
+  stripe,
```

Comment:
This is where the flow gets complicated. Following along with [Stripe's fixed-price subscription example](https://stripe.com/docs/billing/subscriptions/fixed-price), we need to handle a small flowchart of process here.

Not done yet, but trying to arrange this so that some of this could be extracted out of this component.